optimum/exporters/onnx/convert.py

```diff
@@ -40,6 +40,18 @@
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+class ShapeError(ValueError):
+    pass
+
+
+class AtolError(ValueError):
+    pass
+
+
+class OutputError(ValueError):
```

Comment:
A suggestion, feel free to ignore:
```suggestion
class OutputMatchError(ValueError):
```